MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSErrorAttachment.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Maybe create another "physical" folder `Model` under `MobileCenterCrashes` for public model classes. If you do so take a look at other places were this rule may apply for consistency.